src/components/content/Content.jsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import CreateTask from "src/components/tasks/CreateTask";
+import TaskList from "src/components/tasks/TaskList";
+import Button from "src/components/button/Button";
+import { addTask } from "src/store/actions/taskActions";
+import { TEXT_CREATE } from "src/utils/constants/constants";
+import { ALT_ADD_TEXT } from "src/utils/constants/altTexts";
+import { ICON_ADD } from "src/utils/constants/imageSources";
+import styles from "src/components/content/Content.module.scss";
+
+const Content = () => {
+  const [showTask, setShowTask] = useState(false);
+  const [isButtonDisabled, setButtonDisabled] = useState(false);
```

Comment:
Can we maintain button disable attribute from `showTask` state ? Isn't it redundant to maintain an extra state for buttons disable attribute as it seems both(`showTask` & `isButtonDisabled`) are related and updated at the same time in this component.